src/platform-includes/getting-started-install/react-native.mdx

```diff
@@ -8,49 +14,41 @@ npm install --save @sentry/react-native
 yarn add @sentry/react-native
 ```
 
-### Expo
-
-If you are using Expo, see [How to Add Sentry to Your Expo Project](https://docs.expo.io/guides/using-sentry/). This SDK works for both managed and bare projects.
-
-<Note>
-
-Make sure that the version of `@sentry/react-native` matches what `sentry-expo` depends on, [read more on our troubleshooting page](/platforms/react-native/troubleshooting/#expo).
+Run `@sentry/wizard`:
 
-</Note>
+```bash
+npx @sentry/wizard -i reactNative -p ios android
 
-### Linking
+cd ios
+pod install
+```
 
 <Note>
 
-If you are running a project with `react-native` prior to 0.60, you still need to call `react-native link`, otherwise you can skip this step as `react-native` version 0.60 and above does this automatically.
+For React Native 0.68 and older you will have to manually patch the Xcode project. Read more at the [Manual Configuration](/platforms/react-native/manual-setup/manual-setup/) page.
 
 </Note>
 
-Link the SDK to your native projects to enable native crash reporting.
+[Sentry Wizard](https://github.com/getsentry/sentry-wizard) will patch your project accordingly, though you can [setup manually](/platforms/react-native/manual-setup/manual-setup/) if you prefer. You need to do this only once, then the created files can go into your version control system.
 
-```bash
-react-native link @sentry/react-native
-```
+The following changes will be performed by Sentry Wizard:
 
-### Run the Sentry Wizard
+- enable the Sentry React Native Gradle build step for Android to auto upload generated source maps and debug symbols
+- wrap _Bundle React Native code and images_ Xcode project build phase script to upload generated source maps and collect bundled node modules
+- add _Upload Debug Symbols to Sentry_ Xcode project build phase
+- store build credentials in _*ios/sentry.properties*_ and _*android/sentry.properties*_.
+- configure Sentry for the supplied DSN in your _*App.tsx*_ file
+- wrap your _App_ for [touch event tracking](/platforms/react-native/touchevents/) and [automatic performance monitoring](/platforms/react-native/performance/instrumentation/automatic-instrumentation/)
```

Comment:
Is this done by Wizard as well? TIL

Reply:
Actually my bad. It's not. :( But would be cool.